src/value.h

```diff
@@ -20,57 +20,45 @@
 typedef enum {
   T_NULL = 0,
   T_STRING = 0x001,
-  T_INT32 = 0x002,
+  // T_INT32 = 0x002, // unused
```

Comment:
does all values marked as unused were not created, my concern is backward compatibility, we're not expecting those to be seen in old RDBs?